include/dem/grid_motion.h

```diff
@@ -109,14 +111,49 @@ class GridMotion
   move_grid_translational(
     parallel::distributed::Triangulation<dim> &triangulation);
 
+  /**
+   * Carries out translational motion of the triangulation
+   *
+   * @param triangulation Triangulation
+   */
+  void
+  move_grid_due_particles_forces(
+    parallel::distributed::Triangulation<dim> &triangulation);
+
+  /**
+   * Update member information like boundary_forces
+   */
+  void
+  update_parameters_before_motion();
+
+  /**
+   * Update member information like inertia tensor,
+   * center of mass coordinate, etc..
+   */
+  void
+  update_parameters_after_motion();
```

Comment:
Same as above. Specify everything that is updated. Be more descriptive, you must help people to understand what you say.

Reply:
Next commit